spark-bigquery-acceptance-tests/src/main/java/com/google/cloud/spark/bigquery/acceptance/DataprocAcceptanceTestBase.java

```diff
@@ -196,20 +197,17 @@ public void writeStream() throws Exception {
   @Test
   public void testBigNumeric() throws Exception {
     String testName = "test-big-numeric";
-    String pyBaseDir = Paths.get("pythonlib").toAbsolutePath().toString();
-    String zipFileLocation =
-        Paths.get("fatJar").toAbsolutePath().toString() + "/big_numeric_acceptance_test.zip";
+    Path pythonLibTargetDir = Paths.get("../../spark-bigquery-python-lib/target");
+    Path pythonLibZip = AcceptanceTestUtils.getArtifact(pythonLibTargetDir, ".zip");
     String zipFileUri =
         context.testBaseGcsDir + "/" + testName + "/big_numeric_acceptance_test.zip";
 
-    createZipFile(pyBaseDir, zipFileLocation);
-
-    AcceptanceTestUtils.uploadToGcs(
-        getClass().getResourceAsStream("/acceptance/big_numeric.py"),
-        context.getScriptUri(testName),
-        "text/x-python");
+    // AcceptanceTestUtils.uploadToGcs(
+    //     getClass().getResourceAsStream("/acceptance/big_numeric.py"),
```

Comment:
why is this commented?

Reply:
The file is uploaded in `createAndRunPythonJob()`